blockly-core/core/ui/block_space/pan_drag_handler.js

```diff
@@ -59,6 +59,19 @@ Blockly.PanDragHandler = function (blockSpace) {
    * @type {BindData}
    * @private
    */
+  this.wheelKey_ = null;
```

Comment:
Probably more like `wheelEventBindData_` and `mousewheelEventBindData_`.  Was using "key" to refer to the return value of bind operations before we formalized the "BindData" type.